app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/adapters/SmsAdapter.java

```diff
@@ -151,9 +151,9 @@ void displaySmsByType(Sms sms){
             } else {
                 linearLayout.setGravity(Gravity.END);
                 if (Build.VERSION.SDK_INT >= 23) {
-                    cardBubble.setCardBackgroundColor(context.getColor(R.color.cardBackgroundColor));
-                    messageRecieved.setTextColor(context.getColor(R.color.blue_sapphire));
-                    timeDate.setTextColor(context.getColor(R.color.blue_sapphire));
+                    cardBubble.setCardBackgroundColor(context.getColor(R.color.sent_chat_bubble_color));
```

Comment:
Oh I had pulled the cardBackgroundColor from androids default res files, but I'll just make my own and make it snake_case